cmd/livepeer_cli/wizard_eth.go

```diff
@@ -16,3 +16,13 @@ func (w *wizard) setGasPrice() {
 
 	httpPostWithParams(fmt.Sprintf("http://%v:%v/setGasPrice", w.host, w.httpPort), val)
 }
+
+func (w *wizard) signMessage() {
+	fmt.Printf("Enter or paste the message to sign, then press ctrl + d:\n")
+	msg := w.readString()
```

Comment:
This won't be able to handle a multi-line string (which the previous impl in this PR did). An example of a multi-line string is the 3Box auth message:

```
Create a new 3Box profile

-
Your unique profile ID is did:muport:<DID>
Timestamp: <TIMESTAMP>
```

Would be good to double check that the CLI can accept a message with the above format that is copy + pasted from the consent message pop up in the explorer when setting up a 3Box profile:

<img width="607" alt="Screen Shot 2020-01-29 at 6 06 22 PM" src="https://user-images.githubusercontent.com/5933273/73405531-22040980-42c2-11ea-8a2e-39268d1d5479.png">

Reply:
fixed in f811dbd

Reply:
Double checked that the CLI is able to sign the message format provided by 3Box.

Can't really find where I could enter the message and its signature manually to access my profile as a sanity check though. cc @adamsoffer

Reply:
@kyriediculous after you set up your profile, you should see an option to an "add account" underneath the "edit profile" modal. Once you click that you'll see two input fields at the bottom of the form: one to enter the signature and one to ender the address you signed the message with to verify it was signed correctly. If it's valid, you'll see a green message underneath the input letting you know it passed, or red if it failed.

![image](https://user-images.githubusercontent.com/555740/73683505-5c89ef80-4690-11ea-908e-83e21f56bf5d.png)

Reply:
Thanks @adamsoffer 

Looks like it was succesful

![image](https://user-images.githubusercontent.com/22256858/73683802-3cbfef00-46c3-11ea-9b52-8878e54e8915.png)

Reply:
Steps 6-12 in https://forum.livepeer.org/t/how-to-securely-link-your-3box-profile-to-your-orchestrator-account-using-mycrypto/986 describe how to edit an existing profile link an external account by providing a signature and the account that produced the signature.